packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -45,6 +45,7 @@ export 'package:flutter/gestures.dart' show
   TapUpDetails,
   ForcePressDetails,
   Velocity;
+export 'package:flutter/rendering.dart' show RenderSemanticsGestureHandler;
```

Comment:
This export seems odd. Why does the widget layer need to expose a RenderObject?

Reply:
I added it in because `SemanticsGestureDelegate` has a virtual method `void assignSemantics(RenderSemanticsGestureHandler renderObject);`, so anyone who wants to extend it needs this symbol. Do we export symbols like this?

Reply:
we do. this is probably ok.